scenariorun/src/main/kotlin/com/cosmotech/scenariorun/workflow/argo/WorkflowBuilders.kt

```diff
@@ -86,7 +86,7 @@ internal fun buildTemplate(
 
   val template =
       IoArgoprojWorkflowV1alpha1Template()
-          .name(scenarioRunContainer.name)
+          .name(scenarioRunContainer.name.lowercase())
```

Comment:
Why did you put lowercase everywhere?
Is it something needed by solution/project?

Reply:
With the Emissary executor, it's a mandatory to have container's name in lowercase